help_scout-sdk.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'activesupport'
-  spec.add_dependency 'faraday', '1.7.2'
+  spec.add_dependency 'faraday', '1.9.0'
```

Comment:
@mfittko any reason why we don't update this to [1.9.3](https://github.com/lostisland/faraday/blob/main/CHANGELOG.md#193-2022-01-06)?

Reply:
imo no